kbase-extension/static/kbase/js/kbaseNarrative.js

```diff
@@ -228,9 +245,11 @@ define([
                 console.warn('Error removing shortcut "' + shortcut + '"', ex);
             }
             try {
-                Jupyter.notebook.keyboard_manager.edit_shortcuts.remove_shortcut(shortcut);
+                Jupyter.notebook.keyboard_manager.edit_shortcuts.remove_shortcut(
+                    shortcut
+                );
             } catch (ex) {
-                // console.warn('Error removing shortcut "'  + shortcut +'"', ex);
+                // console.warn('Error removing shortcut ''  + shortcut +'"', ex);
```

Comment:
this line won't parse correctly if uncommented

Reply:
This is pre-existing code, I know, but do you know why has this one been commented out but the others left in?

Reply:
I just tested, and it's commented out because of how that whole block is structured. It tries to remove those shortcuts from both the command and edit lists, but they're only in one or the other, so there's 30ish cases of warning spam on every page load if that's uncommented.

That whole function needs an overhaul, imo. Next PR.